### ISSUE TITLE
Question about the results in paper.

### ISSUE THREAD

**Issue by ysoo1133** [KaiyuYue/cgnl-network.pytorch#3]
Hi, thanks for interesting paper :)
I have some questions.

There is no comment in paper whether the results are mean or median or best.
If it is the mean, how many experiments did you execute for the mean?
And are the results in github are included in the mean?

**Comment by kaiyuyue** [comment]
Hi~ 

Really sorry for the delay answering. All the results reported both in the paper and GitHub are best, including the base networks' results.

Honestly, there is a special case on the Fine-Grained dataset of CUB. The ResNet-50 + 1CGNL block did not easily surpass the ResNet-50 + 1NL block first, it just gave me the comparable results, then I did lots of experiments to find the best accuracies. I don't remember how many times now. 

But to the other architectures on the various datasets and for the different tasks as shown on the GitHub, the CGNL block worked well, I achieved all the best results by doing just one experiment for them.

**Comment by KaiyuYue** [comment]
Hi, 

Really sorry for the delay. All the results reported both in the paper and GitHub are best, including the base networks' results.

Honestly, there is a special case on the Fine-Grained dataset of CUB. The ResNet-50 + 1CGNL block did not easily surpass the ResNet-50 + 1NL block first, it just gave me the comparable performances, then I did lots of experiments to find the best accuracies. I don't remember how many times now. 

But to the other architectures on the various datasets and for the different tasks as shown on the GitHub, the CGNL block worked well, I achieved all the best results by doing just one experiment for them.
